app/filemanager/src/queries/update.rs

```diff
@@ -139,25 +139,13 @@ where
     }
 
     /// Verifies that the JSON patch operation is supported.
-    fn verify_patch(
-        patch: Vec<PatchOperation>,
-        current_attributes: &serde_json::Value,
-    ) -> Result<Vec<PatchOperation>> {
-        let check_exists = |path: String, patch: PatchOperation| {
-            let exists = current_attributes.pointer(&path);
-            if exists.is_some() {
-                Err(InvalidQuery("path already exists".to_string()))
-            } else {
-                Ok(patch)
-            }
-        };
```

Comment:
Noting that the code removed here is what allows "add" to replace existing attributes. JSON patch allows this by default, but this code prevented it from replacing an existing path before.